pyproject.toml

```diff
@@ -24,90 +34,15 @@ extend-exclude = '''
 
 '''
 
-[tool.poetry]
-name = "screenpy_playwright"
-version = "0.0.2"
-description = "ScreenPy extension to enable interacting with Playwright."
-authors = ["Perry Goy <perry.goy@gmail.com>"]
-maintainers = ["Marcel Wilson"]
-license = "MIT"
-repository = "https://github.com/ScreenPyHQ/screenpy_playwright"
-documentation = "https://screenpy-playwright-docs.readthedocs.io"
-readme = "README.md"
-classifiers = [
-    "Operating System :: OS Independent",
-    "Programming Language :: Python :: 3 :: Only",
-    "Programming Language :: Python :: 3.8",
-    "Programming Language :: Python :: 3.9",
-    "Programming Language :: Python :: 3.10",
-    "Programming Language :: Python :: 3.11",
-    "Programming Language :: Python :: 3.12",
-    "Development Status :: 5 - Production/Stable",
-    "Intended Audience :: Developers",
-    "Topic :: Software Development :: Quality Assurance",
-    "Topic :: Software Development :: Testing",
-    "Topic :: Software Development :: Testing :: BDD",
-    "Natural Language :: English",
-    "License :: OSI Approved :: MIT License",
-]
-
-
-# It's possible to add optional dependencies with the poetry CLI tool using:
-#   poetry add --optional some_dev_pkg
-# This will set `optional = true` flag in [tool.poetry.dependencies], as seen below
-# But then you need to remember to manually edit the [tool.poetry.extras] dev section
-# and declare the package. This allows `pip install .[dev]` to work as expected
-# Poetry users will need to use the `--extras dev` option rather than the `--with dev`
-# so we dont have two different sets of package versions to update.
-
-[tool.poetry.dependencies]
-python = "^3.8"
-screenpy = ">=4.0.2"
-playwright = ">=1.39.0"
-importlib_metadata = {version = "*", python = "3.8.*"}
-
-# convenience packages for development of screenpy_playwright only
-black = {version = "*", optional = true}
-coverage = {version = "*", optional = true}
-mypy = {version = "*", optional = true}
-pre-commit = {version = "*", optional = true}
-pytest = {version = "*", optional = true}
-ruff = {version = "*", optional = true}
-sphinx = {version = "*", optional = true}
-tox = {version = "*", optional = true}
-
-
-[tool.poetry.extras]
-dev = [
-    "pre-commit",
-    "pytest",
-    "tox",
-]
-dev_all = [
-    "black",
-    "coverage",
-    "mypy",
-    "pre-commit",
-    "pytest",
-    "ruff",
-    "sphinx",
-    "tox",
-]
-test = [
-    "coverage",
-    "pytest",
-]
-
 
 [tool.ruff]
 target-version = "py38"  # minimum supported version
 line-length = 88  # same as Black.
 extend-exclude = [
-    "screenpy_playwright/__init__.py",
-    "screenpy_playwright/__version__.py",
+    "__init__.py",
```

Comment:
I think this one ignores *all* `__init__.py` files, right? I had excluded just the `screenpy_playwright/__init__.py` one on purpose because of the weird formatting i liked to do in there.

But now that i look in it, we can probably just delete the `__init__.py` ignores. I don't see anything weird in there.

Reply:
Actually you know what? I wonder if that's why the `from blah import *` stuff wasn't getting any complaints...

Reply:
It was.  There used to be a few more ruff errors back when we started.  I was able to remove the exclusions.